pkg/lib/cockpit-components-modifications.jsx

```diff
@@ -53,18 +52,17 @@ class ModificationsExportDialog extends React.Component {
 
     handleSelect(event, active_tab) {
         this.setState({ active_tab });
-        clearTimeout(this.tId);
+        this.setState({ copied: false });
     }
 
     copyToClipboard() {
         try {
             navigator.clipboard.writeText(this.props[this.state.active_tab].trim())
                     .then(() => {
                         this.setState({ copied: true });
-                        const timeOutId = setTimeout(() => {
+                        setTimeout(() => {
                             this.setState({ copied: false });
                         }, 3000);
-                        this.tId = timeOutId;
```

Comment:
Cancelling the setTimeout with `clearTimeout` is still valid, in theory one could quickly switch tabs and click copy to clipboard and the previous set timeout would cancel it.

Reply:
Okay